pkg/cluster/config/v1alpha2/types.go

```diff
@@ -69,10 +69,6 @@ const (
 	ControlPlaneRole NodeRole = "control-plane"
 	// WorkerRole identifies a node that hosts a Kubernetes worker
 	WorkerRole NodeRole = "worker"
-	// ExternalEtcdRole identifies a node that hosts an external-etcd instance.
-	// WARNING: this node type is not yet implemented!
-	// Please note that `kind` nodes hosting external etcd are not kubernetes nodes
-	ExternalEtcdRole NodeRole = "external-etcd"
```

Comment:
i don't think we should remove fields from alpha2 as part of the alpha3 introduction.

Reply:
this is a value, not a field, and it is rejected during validation, we're exporting an invalid value and then validating that it is invalid currently 🤷‍♂️